### ISSUE TITLE
Unit Test

### ISSUE THREAD

**Issue by MartinK84** [spinicist/riesling#26]
RIESLING appears to have a Test folder with some automated Unit Tests. I assume that those are run automatically by GitHub?
Is there any way to run the Tests manually from the local command line to check if changes made break anything or just to confirm full functionality? Both the README and CONTRIB documentations unfortunately do not specify anything on this topic.

**Comment by spinicist** [comment]
Yes, there are some unit tests. They are not currently comprehensive but do cover the core gridding/FFT functions. For full functionality, I regard the tutorial notebook as a full Acceptance Test. If that runs, everything should be working.

These tests were previously built by the `prufung` target, but I have pushed an update that changes this to `riesling-tests` for clarity. I have added some notes on these to a new page in the documentation https://riesling.readthedocs.io/en/latest/docs/devel.html
